concept/type/impl/RoleTypeImpl.java

```diff
@@ -164,20 +163,6 @@ public void delete() {
         vertex.delete();
     }
 
-    @Override
-    void validateDelete() {
-        super.validateDelete();
-        if (getInstances().first().isPresent()) {
-            throw exception(TypeDBException.of(INVALID_UNDEFINE_RELATES_HAS_INSTANCES, getLabel()));
-        }
-    }
```

Comment:
Again, moved validation to unset relates. The other option is to move the validation to delete here & for owns.